compiler/qsc_project/tests/projects/basic_manifest/Dependency1.qs

```diff
@@ -0,0 +1,5 @@
+namespace Dependency1 {
+	function First() : String {
```

Comment:
nitpick: Tabs?!? We use spaces everywhere else for our indentation.

Reply:
Does clippy or rustfmt not automatically fix that for us?

Reply:
Not in .qs files ;)